demo/scripts/controls/MainPane.tsx

```diff
@@ -49,6 +54,25 @@ const POPOUT_FEATURES = 'menubar=no,statusbar=no,width=1200,height=800';
 const POPOUT_URL = 'about:blank';
 const POPOUT_TARGET = '_blank';
 
+const emojiPaneProps: EmojiStyle = {
```

Comment:
Suggest not expose these settings for now, just provide a simple emoji plugin with default settings